tests/integration/mci/test_submission_with_errors.py

```diff
@@ -96,7 +59,7 @@ def test_submission_with_errors(self):
         post_data = {
             "action[submit_answers]": "Submit answers"
         }
-        resp = self.client.post(summary_url, data=post_data, follow_redirects=False)
+        resp = self.client.post(mci_test_urls.MCI_0205_SUMMARY, data=post_data, follow_redirects=False)
         self.assertEquals(resp.status_code, 302)
```

Comment:
As discussed the 302 implicitly means it has an error, would it be more explicit to check for 'Page has x errors'